python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -779,20 +779,45 @@ async def create_stream(
             # set the stop_reason for the usage chunk to the prior stop_reason
             stop_reason = choice.finish_reason if chunk.usage is None and stop_reason is None else stop_reason
             maybe_model = chunk.model
+
+            def process_reasoning(reasoning_deltas, content):
+                reasoning_deltas.append(content)
+                return "[Thought]" + content
+
             # First try get content
+            # Separate field acquisition reasoning process
+            if self._model_info["family"] == ModelFamily.R1:
+                if choice.delta.model_extra.get("reasoning_content"):
```

Comment:
This value could be `None` from `model_extra`, so we need to check that before using `len()`